yb-voyager/src/srcdb/oracle.go

```diff
@@ -19,6 +19,9 @@ type Oracle struct {
 	db *sql.DB
 }
 
+var oracleUnsuportedDataTypes = []string{"BLOB", "BFILE", "URITYPE", "XMLTYPE",
```

Comment:
I don't see a few datatypes in this list, such as object types and ref data types, which aren't supported. Are they under some different name, perhaps? 

Also, the list is slightly different for ora2pg vs debezium, so we might need two different lists? 

you can refer to https://docs.google.com/spreadsheets/d/1YMXUXU3qw1tqMeyp1AvRjmkxlcNY5IIsFj_OSu63V-k/edit#gid=0 . This is probably not a complete list, but more about the differences between ora2pg and debezium

Reply:
Oh okay, let me check other types. will update the patch if I could find other types for dbzm than these and create a separate list for them.
yes, for object_types I have added the case for dbzm as there is no specific way to detect UDT as it is created using `CREATE TYPE` so in the `DATATYPE` fetching query it will be `type_name` with which it is created. So, I had to check the `datatype_owner` which will be `source_schema`  in this case. So, for dbzm if any datatype owner is source_Schema I am skipping that column as well but for ORa2pg I can't skip object_types as VARRAYs and NESTED tables which are nothing other than a form of UDT which is supported and I don't think there is any other way to check object type UDTs. So might need to add to a known issue such case of UDT.

Reply:
Got it! Can you add a comment here just mentioning that in addition to these, for dbzm, UDTs are unsupported.

Reply:
okay. will do that